src/js/components/DataTable/doc.js

```diff
@@ -352,6 +352,36 @@ export const themeDoc = {
     type: 'React.Element',
     defaultValue: undefined,
   },
+  'dataTable.pinned.body.background.color': {
```

Comment:
Is it only `background.color` which can be set? What about `opacity`, `image`, etc.? Should the docs be speaking to `dataTable.pinned.body.background` and type would be also include a background type object?

I think the way this is implemented the other background type properties such as image, opacity, etc. all are supported, correct?

Reply:
All background props apply. I will update the site docs to reflect this. Thanks for noting.

Reply:
Updated in recent commits.